### ISSUE TITLE
Making Images Accessible

### ISSUE THREAD

**Issue by RubySpeeders** [RubySpeeders/US-Navy-Project#51]
### User Story

As a user, I want to be able to know that images are selectable, so that I can select the images when using a screen reader.

---

### General Description

The scope of this ticket includes fixing the ability to make the images to be known selectable items on the page. As images are not read as selectable from a screen reader, a user may not know that they can/must click on an image in order to select it.

---

### Technical Steps

- [ ] Replace Me!
- [ ] Replace Me!
- [ ] Replace Me!

---

### Accessibility Acceptance Criteria

- [ ] Tested with voice over / screen reader on

**Comment by RubySpeeders** [comment]
Points: 3

**Comment by cheesehero112** [comment]
We need to make all images "functional images" 

"Functional images are used to initiate actions rather than to convey information. They are used in buttons, links, and other interactive elements. The text alternative for the image should convey the action that will be initiated (the purpose of the image), rather than a description of the image."

[Reference article here](https://www.w3.org/WAI/tutorials/images/functional/)

From what I can tell from the article, all I need to do is to add "alt" property to every image to describe the action that will be initiated when a user clicks the image.

**Comment by cheesehero112** [comment]
After adding alt to each image (line 130 and 160 of selection.tsx), I run chrome's lighthouse extension to test the accessibility. It's currently scoring 91, so not bad, but I will investigate which image it thinks is missing "alt" 

@charlottekies - if you have another way to test if this new alt labels are in fact working, please let me know! 

![Image](https://github.com/RubySpeeders/Drum-Roulette/assets/102322200/bcd4b8c5-d5fa-4a17-9852-2106fba96cac)

**Comment by cheesehero112** [comment]
When I click on which image doesn't have alt, I see the below screenshot, but I'm not sure which one it's talking about...



![Image](https://github.com/RubySpeeders/Drum-Roulette/assets/102322200/6e392557-87b8-4cdc-b6ec-3d056fb197c9)
